### ISSUE TITLE
Unable to create enums with colons and dots [v4.9.0]

### ISSUE THREAD

**Issue by crlssn** [volatiletech/sqlboiler#1117]
### What version of SQLBoiler are you using (`sqlboiler --version`)?
v4.9.0

### What is your database and version
Postgres 12.8

### If this happened at generation time what was the full SQLBoiler command you used to generate your models?
```
sqlboiler --no-tests --no-hooks --add-enum-types
```

### Please provide a relevant database schema so we can replicate your issue
```sql
CREATE TYPE SCOPE AS ENUM ('product.write', 'product:price.write');

CREATE TABLE auth
(
    scope SCOPE NOT NULL PRIMARY KEY
);
```

#### Expected

**File:** `boil_types.go`

```go

type Scope string

// Enum values for Scope
const (
	ScopeProductWrite      Scope = "product.write"
	ScopeProductPriceWrite Scope = "product:price.write'"
)

func AllScope() []Scope {
	return []Scope{
		ScopeProductWrite,
                ScopeProductPriceWrite,
	}
}

func (e Scope) IsValid() error {
	switch e {
	case ScopeProductWrite, ScopeProductPriceWrite:
		return nil
	default:
		return errors.New("enum is not valid")
	}
}

func (e Scope) String() string {
	return string(e)
}

```

#### Actual

**File:** `boil_types.go`

```
// Enum values for  are not proper Go identifiers, cannot emit constants
```

**Comment by crlssn** [comment]
I think I've fixed this here: https://github.com/volatiletech/strmangle/pull/7

**Comment by stephenafamo** [comment]
I'll leave some comments on your PR there.

**Comment by stephenafamo** [comment]
Fixed in `v4.10.1`. Thank you.

**Comment by crlssn** [comment]
Thank you for the quick turnaround!
